### DISCUSSION TITLE
上传OOM了

### DISCUSSION THREAD

**Discussion by zczm2** [messense/aliyundrive-webdav#396] [Q&A] [answered]
有的小米路由器4C刷openwrt，
测试上传一个300M的时候OOM了，
`[ 1277.368692] oom-kill:constraint=CONSTRAINT_NONE,nodemask=(null),global_oom,task_memcg=/,task=aliyundrive-web,pid=2905,uid=0
[ 1277.380035] Out of memory: Killed process 2905 (aliyundrive-web) total-vm:66816kB, anon-rss:32516kB, file-rss:4kB, shmem-rss:0kB, UID:0 pgtables:68kB oom_score_adj:0
[ 1277.766362] oom_reaper: reaped process 2905 (aliyundrive-web), now anon-rss:0kB, file-rss:0kB, shmem-rss:0kB`
挂载参数如下
` aliyundrive-webdav --refresh-token xxxxxx -S 5000000`

**Comment by messense** [comment, marked as answer]
上传要 16M 内存缓冲区，不建议在路由器上部署上传。
